pkg/pod/status.go

```diff
@@ -38,23 +38,23 @@ import (
 	"knative.dev/pkg/apis"
 )
 
-const (
-	// ReasonCouldntGetTask indicates that the reason for the failure status is that the
-	// Task couldn't be found
-	ReasonCouldntGetTask = "CouldntGetTask"
-
+// Aliased for backwards compatibility; do not add additional TaskRun reasons here
+var (
 	// ReasonFailedResolution indicated that the reason for failure status is
 	// that references within the TaskRun could not be resolved
-	ReasonFailedResolution = "TaskRunResolutionFailed"
-
+	ReasonFailedResolution = v1.TaskRunReasonFailedResolution.String()
```

Comment:
why do we need a .String() here?

Reply:
The reasons previously were just strings while the v1.TaskRunReasons* are of type TaskRunReason.

Reply:
Yeah, it looks like only `ReasonFailedResolution` is attached with `.String()` before. Now it is consistent. Thanks.